spec/features/user_updates_profile_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+# Acceptence Criteria
+# [✓] I can navigate to an update page for my profile
+# [✓] My profile is populated with accurate information
+
+feature "As an authenticated user
+I want to update my information
+So that I can keep my profile up to date" do
+  scenario "authenticated user clinks link to update information" do
+    user = FactoryGirl.create(:user)
+    visit '/'
+    click_link("Sign In")
+    fill_in 'Email', with: user.email
+    fill_in 'Password', with: user.password
+    click_button('Log in')
+    click_link 'Edit Profile'
+    expect(page).to have_content("Edit User")
+  end
+end
```

Comment:
You should add some more code to this test. Have the user change information and check that the data persists.